contract-message-test/src/transaction_utils.sw

```diff
@@ -0,0 +1,17 @@
+library transaction_utils;
```

Comment:
hm, I thought I just commented on this above. Perhaps this should be extracted into a library project and imported rather than duplicated into multiple projects?

Reply:
They are custom tailored to each sections need. There was an attempt to add them to the sway std lib, but that implementation included extra checks to avoid hard reverts. It was decided to keep these ultra light (code size wise) implementations for use by these predicate related contracts that need code size as small as possible.